docs/SETUP.md

```diff
@@ -61,6 +61,15 @@ We're on Webpacker, which requires an additional setup step, but which lets us w
 * Install Yarn locally (`brew install yarn`, or the [setup instructions](https://yarnpkg.com/en/docs/install)).
 * Install JS packages: `yarn install`
 
+### Then, Postgres dependencies
+We use Postgres around these parts. Installation will differ based on your OS.
+
+* Install Postgres
+  * MacOS: `brew install postgres`
+  * Ubuntu: install `postgresql`, `libpq-dev`, and `postgresql-client`
+  * Other [linux instructions](https://www.postgresql.org/download/) (server and developer libraries needed)
+* Run the command `rails db:create && rails db:migrate` to set up the tables
```

Comment:
Can you add the `docker-compose` version of these instructions above (line 20).

Reply:
done!